### ISSUE TITLE
Doesn't log if MEDIA_URL is not set

### ISSUE THREAD

**Issue by Mariama4** [vishalanandl177/DRF-API-Logger#101]
Thanks for the handy library, but I noticed a bit of strange behavior (maybe it should be like that), why if "MEDIA_URL" is not set, it treats it as "/" and not "getattr(settings, 'MEDIA_URL', '/media/')" as it is written in the library middleware.

As a result, it treats any request as a media request and skips logging it.

![Снимок экрана 2024-10-03 в 11 13 19](https://github.com/user-attachments/assets/d082dd2f-1b31-4505-97d3-09dbbe9500f7)
![Снимок экрана 2024-10-03 в 11 14 18](https://github.com/user-attachments/assets/7ede012b-505b-459d-a69b-a5a20ed7e845)
![Снимок экрана 2024-10-03 в 11 15 39](https://github.com/user-attachments/assets/f57a183f-1303-4c92-b988-4da7e4fa1507)

Django version - 5.1
Python version - 3.11

**Comment by t4d** [comment]
Indeed, this will not log anymore until you have a MEDIA_URL in your settings.

**Comment by N1K1TAS95** [comment]
I had the same issue.

This is the function in `APILoggerMiddleware` class that checks if the path is for static or media files:
``` python
def is_static_or_media_request(self, path):
    static_url = getattr(settings, 'STATIC_URL', '/static/')
    media_url = getattr(settings, 'MEDIA_URL', '/media/')
    
    return path.startswith(static_url) or path.startswith(media_url)
```

So if your `MEDIA_URL` or `STATIC_URL` is default to `/` al requests will be ignored.

Maybe it should check if contains condition like so:
``` python
def is_static_or_media_request(self, path):
    static_url = getattr(settings, 'STATIC_URL', '/static/')
    media_url = getattr(settings, 'MEDIA_URL', '/media/')

    return static_url in path or media_url in path
```

**Comment by vishalanandl177** [comment]
This issue is fixed https://github.com/vishalanandl177/DRF-API-Logger/pull/112.
It will be available in the next release.
